README.md

```diff
@@ -69,21 +70,28 @@ Alternatively, install [manually](/docs/Install_addon_manually.md) via Blender s
 >  Only relevant if you are wanting to run the tests.
 > The tests make use of the [pytest-blender plugin](https://github.com/mondeja/pytest-blender#pytest-blender), which has `pytest` and other packages as dependencies (e.g. `pytest-cov`). These need to be installed in the site-packages directory of Blender's Python. The pytest-blender repo provides some guidance for this [here](https://github.com/mondeja/pytest-blender#usage). It is important to make sure you use Blender's Python interpreter and Blender's pip when installing `pytest` and its dependencies. Below are some tips on how to do this in Linux and MacOS.
 
-> **Linux**
->  An easy way to install these dependencies correctly in Linux is to run the following code in [Blender's Python scripting window](https://docs.blender.org/api/current/info_quickstart.html):
-> `import pip `
-`pip.main(["install", "pytest", "--user"])`
-`pip.main(["install", "pytest-cov", "--user"])`
-`pip.main(["install", "pytest-blender", "--user"])`
-
-
-> [!CAUTION]
-> **MacOS**
-> Installing Blender in an isolated environment may be preferable. We occasionally found some issues with the `pytest` installation for Blender, where it seemed that running pytest from a different environment would use the `pytest` installed in the Blender environment. To troubleshoot this, it may be useful to run `which pytest`from the active environment: if all is working fine, that should point to the `pytest` of the active environment. One can also force to run the `pytest` installation of the active environment by running `python -m pytest`.
->  The following steps were needed to install these dependencies correctly on MacOS:
-> - `get-pip.py` downloaded (this step may not be needed since newer versions of Blender have pip installed already in the Blender python)
-> - Changing Mac permissions to grant full disk access from where you're running pytest i.e. VS code or terminal
-> - `/Applications/Blender.app/Contents/Resources/3.4/python/bin/python3.10 -m pip install pytest -t "/Applications/Blender.app/Contents/Resources/3.4/python/lib/python3.10/site-packages"` the target flag -t makes sure that the installation ends up in the correct place
+**Linux**
+
+An easy way to install these dependencies correctly in Linux is to run the following code in [Blender's Python scripting window](https://docs.blender.org/api/current/info_quickstart.html):
+```
+import pip
+pip.main(["install", "pytest", "--user"])
+pip.main(["install", "pytest-cov", "--user"])
+pip.main(["install", "pytest-blender", "--user"])
+```
+
+
+**MacOS**
+
+The following steps were needed to install these dependencies correctly on MacOS:
+```
+pip install pytest-blender
```

Comment:
Should we suggest running these commands in a virtual environment since it's generally good practice? 

Also, I had to `pip install pytest-cov` too and add `~/.local/bin` to my path to make it work 😬